### ISSUE TITLE
Activate navigation from home screen

### ISSUE THREAD

**Issue by fbnlsr** [impshum/Tutor-Piles#2]
When launching the presentation, one has to use the menu to manually select the first slide with a click, and then the keyboard navigation is available.

I feel like keyboard navigation should be available from the get go and we should be able to move from the index page to the first slide.

**Comment by impshum** [comment]
The index.md is the splash page. I intend to keep it that way for now as you can leave it up on the screen when not teaching displaying the course/class info etc.

**Comment by fbnlsr** [comment]
Oooh I just realized that you can handle MULTIPLE slideshows within the same `content` directory! That's actually a great feature!
